mlflow/entities/span_status.py

```diff
@@ -52,7 +52,11 @@ def __post_init__(self):
                 )
 
     def to_otel_status(self) -> trace_api.Status:
-        """Convert our status object to OpenTelemetry status object."""
+        """
+        Convert our status object to OpenTelemetry status object.
```

Comment:
Can we replace `our` with something else?

Reply:
Agree, we can replace it to py class ref.